### PR TITLE
Isolate VM support in V1

### PR DESCRIPTION
**Pull request by AkhileshK8y**
## Isolate VM support in Transformer v1

> This PR is aimed to add on demand isolate VM support for
  multi-tenant architecture.
> We introduce a environment variable ON_DEMAND_ISOLATE_VM
  which when valuated to a non zero number means on demand
  isolated VMs are to be used. Pooled way other wise.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: util/customTransformer-v1.js [hunk 1]
```diff
@@ -2,8 +2,18 @@ const ivm = require("isolated-vm");
 const stats = require("./stats");
 
 const { getPool } = require("./ivmPool");
+const { getFactory } = require("./ivmFactory");
+const logger = require("../logger");
 
-
+/*
+Transform VM aquire mode is
+ Pooled if env variable ON_DEMAND_ISOLATE_VM = 0
```
Comment (lokey) [inline util/customTransformer-v1.js:10]: Can you make the env variable boolean instead of 1 and 0?

Comment (AkhileshK8y) [reply util/customTransformer-v1.js]: Done.

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by gane5hvarma — APPROVED** [review]

File: __tests__/user_transformation.test.js [hunk 2]
```diff
@@ -24,264 +24,275 @@ const randomID = () =>
...
-          const filteredEvents = events.map(event => {
-            return event;
-          });
-            return filteredEvents;
-          }
-          `
-    };
-    fetch.mockResolvedValue({
-      json: jest.fn().mockResolvedValue(respBody)
+const possibleEnvs = ["true", "false"]
```
[the remark this answers is not in the document]
Comment (AkhileshK8y) [reply __tests__/user_transformation.test.js]: Done..

**Pull request merged**